### DISCUSSION TITLE
How to call C# functions with parameters from blueprints

### DISCUSSION THREAD

**Discussion by burakakca** [nxrighthere/UnrealCLR#265] [Help]
Hi,

I have a question about UnrealCLR Library;

I want to call C# function via Blueprint. You have "FindManagedFunction" & "ExecuteManagedFunction" to execute managed c# functions. I am not sure if it supports function parameters and getting return results from the function.

How can I execute C# .net managed function by passing function parameters and  how can I get returned results from the C# function?  Not UObjects !

Thank you

**Comment by nxrighthere** [comment]
Please, see https://github.com/nxrighthere/UnrealCLR/discussions/153.
